src/config.rs

```diff
@@ -1,9 +1,9 @@
 use fastly::http::header::HeaderName;
 use fastly::http::header::{CONTENT_LENGTH, CONTENT_TYPE};
 
-/// This should be changed to match the name of your storage backend. See the the `Hosts` section of
+/// This should match the name of your storage backend. See the the `Hosts` section of
 /// the Fastly WASM service UI for more information.
-pub(crate) const BACKEND_NAME: &str = "mock-s3.edgecompute.app";
+pub(crate) const BACKEND_NAME: &str = "bucket_origin";
```

Comment:
This would cause the starter-kit to break, but shouldn't we provide a value that means the starter-kit works by default?

Reply:
backend name is not a hostname, so I think it's better that it doesn't look like one